### ISSUE TITLE
Synchronize editor and preview pane scrolling

### ISSUE THREAD

**Issue by DaveJarvis** [DaveJarvis/scrivenvar#31]
**Replicate**
1. Scroll the editor pane using the scrollbar.

**Expected Results**
The preview pane scrolls in lock-step with the editor pane.

**Actual Results**
The preview pane does not scroll and must be scrolled independently.

**Comment by DaveJarvis** [comment]
This requires finding the percentage scrolled in the editor and moving the scroll panel in the HTML preview panel (via WebView) to the same percentage.

``` .java
Var estimatedScrollYProperty = scrollPane.estimatedScrollYProperty();
```

A bidirectional lock probably isn't possible.

**Comment by DaveJarvis** [comment]
Fixed in 1.6.0.
